data/yaml/subdivisions/sa.yaml

```diff
@@ -42,9 +42,9 @@
     maxlatitude: 24.659185
     latitude: 24.466667
     longitude: 39.6
-- name: Ash Sharqiyah
+- name: Eastern
```

Comment:
Didn't the customer ask for "Ash Sharqiyah (Eastern)" ?

Reply:
@cchigurupati We already show `Northern` instead of `al-Hudud ash-Shamaliyah`. So went with just `Northern`. Customer originally requested for `Eastern` to be available. However, When we suggested to look for `Ash Sharqiyah` then customer responded back with showing `Ash Sharqiyah (Eastern)`.

Reply:
@cchigurupati @asif-srti 
If we are to change `Ash Sharqiyah` -> `Ash Sharqiyah (Eastern)` then customer may complain that `Northern` region should be `Al-Hudud Ash-Shamaliyah`. 

Please advise further.